packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -272,6 +298,13 @@ export class WaferMap extends FoundationElement {
 
     private diesChanged(): void {
         this.waferMapUpdateTracker.track('dies');
+        this.renderStrategy = this.dies === undefined ? 'worker' : 'main';
+        this.waferMapUpdateTracker.queueUpdate();
+    }
+
+    private diesTableChanged(): void {
+        this.waferMapUpdateTracker.track('dies');
```

Comment:
Should this be `this.waferMapUpdateTracker.track('diesTable');`?

Reply:
I was aiming to replace the dies step in the update cycle, so I thought that duplicating it will not be necessary